src/reversion/revisions.py

```diff
@@ -8,12 +8,17 @@
 from weakref import WeakValueDictionary
 import copy
 
+try:
+    from django.apps import apps
+except ImportError:  # For Django < 1.7
+    from django.db import models as apps
+
```

Comment:
This feels like a fallback that isn't correct, as the two modules aren't equivalent. We should really be specifically importing get_models() and get_apps() using from django.(apps/models) import get_models, get_apps.